doc/source/data/dataset-ml-preprocessing.rst

```diff
@@ -90,6 +90,7 @@ Other preprocessing operations require global operations, such as groupbys and g
 
```

Comment:
@c21 Can you update line 83, which says "Just like other transformations, grouped/global aggregations are executed *eagerly* and block until the aggregation has been computed."? This is no longer true: all "other transformations" are lazy, grouped aggregations are lazy, and global aggregations are eager (since they materialize a scalar).

https://github.com/ray-project/ray/blob/7f5925bbf3799e1369b6185f46be7141abadc9c0/doc/source/data/dataset-ml-preprocessing.rst?plain=1#L83

Reply:
@clarkzinzow - yeah good catch, updated.